packages/core/src/profile/client/postPhoneNumberValidations.js

```diff
@@ -0,0 +1,23 @@
+import client, { adaptError } from '../../helpers/client';
+
+/**
+ * Method responsible for sending a phone token to the specified phone number.
+ *
+ * @function postPhoneNumberValidations
+ * @memberof module:profile/client
+ *
+ * @param {object} [data] - Object containing the phone number.
```

Comment:
I believe this parameter is not optional, so it should be `data` instead of `[data]` (valid for other clients as well)